ui/src/views/actions/thunks.ts

```diff
@@ -112,8 +115,50 @@ export const getViewForTimeMachine = (
         view,
       })
     )
+  } catch (error) {
+    console.error(error)
+    dispatch(notify(copy.getViewFailed(error.message)))
+    dispatch(setView(cellID, RemoteDataState.Error))
+  }
+}
+
+export const setQueryResultsByQueryID = (queryID: string) => (
+  dispatch,
+  getState: GetState
+): Promise<void> => {
+  try {
+    const state = getState()
+    const files = state.data.queryResultsByQueryID[hashCode(queryID)]
+    if (files) {
+      dispatch(setQueryResults(RemoteDataState.Done, files, null, null))
+      return
+    }
+    dispatch(executeQueries())
+  } catch (error) {
+    // if the files don't exist in the cache, we want to execute the query
+    console.error(error)
```

Comment:
if i'm understanding this correctly, this logs an error on cache misses? seems noisy - if that's the case i think we should just swallow this error and not log it.

Reply:
Good point.